core/src/main/scala/org/apache/spark/rdd/JdbcRDD.scala

```diff
@@ -34,7 +34,7 @@ private[spark] class JdbcPartition(idx: Int, val lower: Long, val upper: Long) e
 
 // TODO: Expose a jdbcRDD function in SparkContext and mark this as semi-private
 /**
- * An RDD that executes an SQL query on a JDBC connection and reads results.
+ * An RDD that executes a SQL query on a JDBC connection and reads results.
```

Comment:
Some people write "an" here because they pronounce SQL as "ess-cue-ell" not "sequel", but, I say "sequel" and that was actually the origin of the name, so this is fine. Structured Query Language is a backronym.